deploy/docker/Dockerfile

```
@@ -41,15 +41,19 @@ RUN gem install \
         fluent-plugin-rewrite-tag-filter:2.2.0 \
         fluent-plugin-prometheus:1.6.1
 
-# Install Terraform
-ENV TERRAFORM_VERSION 0.12.26
+# Install Terraform ...
+ENV TERRAFORM_VERSION 0.12.29
 RUN mkdir /tmp/terraform \
  && curl https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -o /tmp/terraform/terraform.zip \
  && cd /tmp/terraform \
  && unzip terraform.zip \
  && mv terraform /bin/terraform \
- && chmod +x /bin/terraform \
- && rm -rf /tmp/terraform
+ && chmod +x /bin/terraform
+
+# ... and providers
+COPY main.tf /tmp/terraform
```

```suggestion
COPY helm/sumologic/conf/setup/main.tf /tmp/terraform
```

Shouldn't this work?

---

No because docker build context is being set to `deploy/docker` 

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/75ca4c12121a24840da02a634d57fb75ff18a1d5/ci%2Fbuild.sh#L129-L135

We might change that obviously if we're ok with sending slightly bigger context to docker daemon.

Shouldn't be a problem to be honest

```
du -sh *
4.0K    README.md
 20K    docker
184K    docs
1012K   helm
4.0K    test
```

---

Let's keep it as it is. Same as for:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/75ca4c12121a24840da02a634d57fb75ff18a1d5/ci%2Fbuild.sh#L52